layouts/partials/head.html

```diff
@@ -0,0 +1,45 @@
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+  <title>{{ .Title }}</title>
+
+  {{ $style := resources.Get "scss/index.scss" }} {{ $options := dict
+  "transpiler" "dartsass" }} {{ $style := $style | resources.ToCSS $options}}
+
+  <link rel="stylesheet" type="text/css" href="{{ $style.RelPermalink }}" />
+
+  <link rel="preconnect" href="https://fonts.gstatic.com" />
+  <link
+    href="https://fonts.googleapis.com/css2?family=Montserrat:wght@500;600;700&family=Source+Code+Pro:wght@700&display=swap"
+    rel="stylesheet"
+  />
+
+  <link
+    rel="alternate"
+    type="application/rss+xml"
+    href="{{.Site.BaseURL }}feed.xml"
+    title="RSS Feed for dev.maxmind.com"
+  />
+
```

Comment:
Do we want to also add the scripts that were added in this PR?

```
  {{ $search := resources.Get "js/search.ts" | js.Build }}
  {{ $searchResult := resources.Get "js/search-result.ts" | js.Build }}
  {{ $systemStatus := resources.Get "js/system-status.ts" | js.Build }}
  {{ $js := slice $search $searchResult $systemStatus | resources.Concat "js/bundle.js" }}
```

Reply:
Nah, I might just delete the files and we can add them later when the associated stories are picked up